pkgs/development/libraries/togl/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib
+, fetchurl
+, libGL
+, tcl
+, tk
+, libX11
+, libXmu
+}:
+
+tcl.mkTclDerivation rec {
+  pname = "togl";
+  version = "2.0";
+
+  src = fetchurl {
+    url = "mirror://sourceforge.net/projects/togl/files/Togl/${version}/Togl${version}-src.tar.gz";
+    hash = "sha256-t9SpC7rTrKYY1QXumef9j7BMgp9jIx3aI2D1V7o/dhA=";
+  };
+
+  postPatch = ''
+    sed -i "s|\".*/generic/tclInt.h\"|\"${tcl}/include/tclInt.h\"|" configure
+    sed -i "s|\".*/generic/tkInt.h\"|\"${tk.dev}/include/tkInt.h\"|" configure
+  '';
+
+  configureFlags = [
+    "--with-tcl=${tcl}/lib"
+    "--with-tk=${tk}/lib"
+  ];
+
+  buildInputs = [
+    tcl
+    tk.dev
+    libGL
+    libX11
+    libXmu
+  ];
```

Comment:
Are all these things needs to be in `buildInputs` ? Shouldn't they belong to `nativeBuildInputs` ?

Reply:
I am not pretty sure...
How can I chack it?

Reply:
If the derivation is a library, there's a lot of chance it should go in `nativeBuildInputs`. If it's a binary, then in `buildInputs`.